src/StarcounterClientFiles/bower.json

```diff
@@ -7,10 +7,11 @@
     "imported-template": "^3.1.1",
     "palindrom-redirect": "~1.0.0",
     "juicy-redirect": "Juicy/juicy-redirect#~0.6.0",
-    "bootswatch": "thomaspark/bootswatch#~3.3.5",
     "starcounter-include": "^4.2.0",
     "polymer": "^2.0.0",
-    "webcomponentsjs": "^1.0.0"
+    "webcomponentsjs": "^1.0.0",
+    "Underwear.css": "underwear.css#^0.0.1",
```

Comment:
Underwear should be with a small `u`, otherwise it installs in the directory `Underwear.css` which does not match the path in `app-shell.html`.

This can be fixed in the next release, since Windows forgives us that.

Reply:
Have you installed it via `bower install underwear.css --save`? If so it should look like `"underwear.css": "^0.0.1",`

Reply:
It is `Underwear.css` because it was defined in that way here: https://github.com/Starcounter/Underwear.css/blob/c595b74a49e6e538af4bcba6f1c9c64289566fb2/bower.json#L2